addons/html_builder/static/src/plugins/carousel_option.xml

```diff
@@ -82,4 +82,41 @@
     </button>
```

Comment:
Since you are doing this carousel in particular, I don't know if you noticed but in the snippet dialog, the card "stretches" and is too big, maybe you can fix that while you are at it ? :slightly_smiling_face: (surely a CSS issue)
![image](https://github.com/user-attachments/assets/adb99c9f-1a1a-4d9f-b598-d0f0ba8f5829)